google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/FlexibleFacetEditor.java

```diff
@@ -278,28 +289,59 @@ public void onFacetInitialized(@NotNull Facet facet) {
 
     private final Project project;
     private final String fileName;
-    private final TextFieldWithBrowseButton filePicker;
+    private final TextFieldWithBrowseButton directoryPicker;
     private final Consumer<Path> configFileGenerator;
+    private final boolean isDirectory;
     // Used to refresh the warnings.
     private final Runnable configurationValidator;
 
+    /**
+     * Constructor
```

Comment:
I don't think this is proper javadoc /  google format. I'd add a sentence (and I think it has to end in a period).